assets/queries/cloudFormation/default_security_groups_deny_traffic/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "default_security_groups_deny_traffic",
+  "queryName": "Default Security Groups Deny Traffic",
```

Comment:
The queryName should define the problem, in this case, something like: "Default SecurityGroups Allow Traffic"